code/nxt/src/display_data.c

```diff
@@ -2,48 +2,75 @@
 
 #include "nxt.h"
 #include "display_data.h"
+#include "data_receive.h"
 #include "movement.h"
 
 void display_target_location(T_TARGET_LOCATION loc)
 {
-  display_goto_xy(0, 1);
-  display_string("t: "); // timestamp
-  display_goto_xy(4, 1);
-  display_int(loc.timestamp, 7);
+    display_clear(0);
+    display_update();
+   
+    /* SHOW COMPILATION TIME TO CHECK IF RIGHT VERSION IS RUNNING */
```

Comment:
Don't know if it's just me, but I think the compilation time should just be printed on the init screen, kind of like a version.
It is not so relevant, once we start.

Reply:
ALSO, WHY ARE WE YELLING?? :joy: